README.md

```diff
@@ -1,5 +1,7 @@
 Authors Haven - A Social platform for the creative at heart.
 =======
+[![Build Status](https://travis-ci.org/mirest/ah-haven-space-sprinters.svg?branch=develop)](https://travis-ci.org/mirest/ah-haven-space-sprinters) [![Coverage Status](https://coveralls.io/repos/github/mirest/ah-haven-space-sprinters/badge.svg?branch=develop)](https://coveralls.io/github/mirest/ah-haven-space-sprinters?branch=develop) [![Codacy Badge](https://api.codacy.com/project/badge/Grade/396fc4ef974d44a5a6045406ebbe4130)](https://www.codacy.com/app/ManuelDominic/ah-haven-space-sprinters?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=mirest/ah-haven-space-sprinters&amp;utm_campaign=Badge_Grade)
```

Comment:
@ManuelDominic , only add bagdes for codeclimate and coveralls here

Reply:
@kimbugp, i only add the two while at the same time just shifted @Rhytah's badges from up next to the other badges.